enso/experiment/NB.py

```diff
@@ -25,6 +28,8 @@ def predict(self, dataset):
         probabilities = self.active_model.predict_proba(dataset)
         return pd.DataFrame({label: probabilities[:, i] for i, label in enumerate(labels)})
 
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "not PlainTextFeaturizer")])
```

Comment:
Maybe consider something like "Featurizer": "not PlainTextFeaturizer" as input rather than list of tuples?